### PR TITLE
Update landing page design

### PR DESCRIPTION
**Pull request by jamiemd**
screen shot 2018-07-26 at 7 28 35 pm

### CODE REVIEW

File: templates/basic/content/src/pages/home.js [hunk 1]
```diff
@@ -1,5 +1,72 @@
...
+  marginTop: '20px',
+  backgroundColor: 'white',
+  ':hover': {backgroundColor: '#f0f8fa'},
+  border: '10px solid #4db5d9',
+  borderRadius: '50%',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+});
+// const InnerCircle = styled('div', {height: '150px', width: '150px', backgroundColor: 'white', ':hover': {backgroundColor: '#f0f8fa'}, border: '2px solid #4db5d9', borderRadius: '50%', display: 'flex', justifyContent: 'center', alignItems: 'center'});
```
Comment (KevinGrandon) [inline templates/basic/content/src/pages/home.js]: nit: remove this.

File: templates/basic/content/src/pages/pageNotFound.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import {NotFound} from 'fusion-plugin-react-router';
 
 const PageNotFound = () => (
   <NotFound>
-    <div>404</div>
+    <div />
```
Comment (KevinGrandon) [inline templates/basic/content/src/pages/pageNotFound.js]: nit: probably leave this file alone for now, or update the messaginng.

File: templates/basic/content/src/pages/styles.js [hunk 3]
```diff
@@ -4,5 +4,6 @@ import React from 'react';
 import {styled} from 'fusion-plugin-styletron-react';
 
 const Panel = styled('div', {background: 'silver'});
+
```
Comment (KevinGrandon) [inline templates/basic/content/src/pages/styles.js]: nit: revert this change.

File: templates/basic/content/src/pages/home.js [hunk 4]
```diff
@@ -1,5 +1,72 @@
...
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  textAlign: 'center',
+  height: '100%',
+});
+
+const Home = () => (
+  <FullHeightDiv>
+    <style>
```
Comment (KevinGrandon) [inline templates/basic/content/src/pages/home.js]: Probably should use helmet for this, but we were getting an error with prepared() for some reason, will investigate.

**Comment by lhorie** [comment]
Nice!
